pkg/restore/pod_action_test.go

```diff
@@ -169,17 +169,23 @@ func TestPodActionExecute(t *testing.T) {
 		t.Run(test.name, func(t *testing.T) {
 			action := NewPodAction(arktest.NewLogger())
 
-			res, warning, err := action.Execute(test.obj, nil)
-
-			assert.Nil(t, warning)
+			res, err := action.Execute(&RestoreItemActionExecuteInput{
+				Item:           test.obj,
+				ItemFromBackup: test.obj,
+				Restore:        nil,
+			})
 
 			if test.expectedErr {
 				assert.NotNil(t, err, "expected an error")
```

Comment:
We could add a `continue` here, and then we wouldn't need the `else` that follows (but we'd keep the assert-no-error check), and we also wouldn't need to check `if !test.expectedErr` again below.

Reply:
adding `continue` tells me `you're not in a loop` ;)
I added `return`